Project-I-Social-Card/social-card/src/components/HeaderComponents/ImageThumbnail.js

```diff
@@ -1,2 +1,9 @@
 import React from 'react';
 import './Header.css';
+
+const ImageThumbnail = props => {
```

Comment:
since you are not using props or passing them anywhere through this component, there is no need to include them in your function declaration.
you could also write this as 
`const ImageThumbnail = () => {}`